### PR TITLE
Build account page

### PR DESCRIPTION
**Pull request by ClementChaumel**
## Done

- Create an empty /account page

## QA

- go to https://ubuntu-com-10004.demos.haus
- sign in
- check that https://ubuntu-com-10004.demos.haus/account exists and displays your email

## Screenshots

image

### CODE REVIEW

**Comment by elioqoshi** [comment]
I took the liberty to QA+1 and UX+1 as well. LGTM!

**Comment by anasereijo** [comment]
LGTM

**but** on smaller screens there's extra space on the left side

Screenshot 2021-07-12 at 11 34 53

**Comment by ClementChaumel** [comment]
@anasereijo PTA!

File: webapp/advantage/views.py [hunk 1]
```diff
@@ -737,6 +737,16 @@ def accept_renewal(renewal_id, **kwargs):
...
+@advantage_checks(check_list=["is_maintenance", "view_need_user"])
+def account_view(**kwargs):
+    email = flask.session["openid"]["email"]
+
+    return flask.render_template(
+        "account/index.html",
+        email=email,
+    )
+
+
```
Comment (albertkol) [inline webapp/advantage/views.py:749]: Beautiful written code. :+1: Well done, Clem!

**Review by albertkol — APPROVED** [review]
:+1:

**Pull request merged**